.ci/install_crio.sh

```diff
@@ -39,6 +39,11 @@ if [ "$ID" == "fedora" ]; then
 	fi
 fi
 
+echo "Get CRI Tools"
+crictl_version=$(get_test_version "externals.critools.version")
+wget -qO- https://github.com/kubernetes-sigs/cri-tools/releases/download/$crictl_version/crictl-$crictl_version-linux-"$(${cidir}/kata-arch.sh -g)".tar.gz \
+	| tar xfz - -C /usr/bin
```

Comment:
seems that this needs to be `sudo tar ...`

Reply:
Makes sense, I change it.

Reply:
thanks